recording-oracle/src/modules/auth/refresh-token.entity.ts

```diff
@@ -11,7 +10,6 @@ import { DATABASE_SCHEMA_NAME } from '@/common/constants';
 import type { UserEntity } from '@/modules/users';
 
 @Entity({ schema: DATABASE_SCHEMA_NAME, name: 'refresh_tokens' })
-@Index(['userId'], { unique: true })
```

Comment:
This will be implicitly added by `@OneToOne` as unique constraint

Reply:
If we have both `@Index` and `@OneToOne`, then we will have two indexes: explicit one created by `@Index` and implicit one, created by `UNIQUE CONSTRAINTS` added by `@OneToOne`, so probably cleaner to keep it as is